pkgs/development/python-modules/asteroid-filterbanks/default.nix

```diff
@@ -36,7 +35,6 @@ buildPythonPackage rec {
     coverage
     librosa
     numpy
-    pre-commit
     pytest
     scipy
     torch
```

Comment:
Looks like that more of the linting modules could be removed, see https://github.com/asteroid-team/asteroid-filterbanks/blob/v0.4.0/setup.py#L37.

While you are on it...could you please move `librosa` and `scipy` into `optional-dependencies`?

Reply:
Got it, and checked that everything still builds

Reply:
`black` and `coverage` are not runtime dependencies. `typing-extensions` is one (https://github.com/asteroid-team/asteroid-filterbanks/blob/v0.4.0/setup.py#L37). I guess that there was no failure because `typing-extensions` is pulled-in by other module.

Also, it looks like that `pytest` is misplaced.

```text
  nativeCheckInputs = [
    pytestCheckHook
  ];
```

Reply:
> black and coverage are not runtime dependencies.

So you're suggesting that these be moved to `buildInputs`?

> typing-extensions is one

and this be added to `propagatedBuildInputs`?

> Also, it looks like that pytest is misplaced.

I'm not sure what you mean by this.